templates/zerver/help/quote-and-reply.md

```diff
@@ -3,6 +3,16 @@
 You can quote a previous message, either replying within a topic or to
 start a new topic.
 
+The first line of the reply contains a [permanent link][link-to-message]
+to the quoted message. Zulip automatically turns mentions in the quoted text
+into [silent mentions](/help/mention-a-user-or-group#silently-mention-a-user)
+to avoid unnecessarily mentioning someone twice.
+
+Note that you can use [quote blocks](/help/format-your-message-using-markdown#quotes)
```

Comment:
Let's make this note into a tip, and then we can leave off the "Note that" part.